### DISCUSSION TITLE
Bluetooth navigation audio while radio is playing

### DISCUSSION THREAD

**Discussion by quicksi1vr** [BimmerGestalt/AAIdrive#814] [Q&A] [answered]
Hello community, looking for a possible solution for playing navigation voice commands from phone while the radio is on.
So, I use Google Maps on my phone (which is on a phone mount) for navigation. I mostly listen to the radio, so with AAIdrive, my phone automatically pairs with the car's bluetooth, and I don't get the voice guidance from the navigation app, as my phone transmits all audio via bluetooth now. BNut the car audio doesn't play this back as I'm listening to the radio. Any ideas (or settings I'm missing) to overlay the navigation audio over the car's speakers, irrespective of what media I'm listening to?
FYI, it works as expected when I'm listening to something from a phone app though (Spotify, Podcasts etc).
Thanks in advance!

**Comment by hufman** [comment, marked as answer]
Good evening!
Unfortunately, I don't think this can be done easily. I believe there's protocol support for the phone to request audio focus temporarily, but I have not figured out how this works. Even then, the navigation announcements would probably be difficult to detect and act on. For example, in order to receive audio focus changes, to detect when the announcements trigger audio ducking, the app must hold audio focus at that time.
It would definitely be handy! If anyone thinks of anything, it would probably be a relatively quick addon to build.

**Reply by beer-matt** [reply]
The app [Speedtrap Alert](https://play.google.com/store/apps/details?id=com.aleprojects.speedtrapalert) has an option to use hands-free mode for alerts (with a secondary option to not do this if music is playing) ; when enabled this appears similar to a call on the phone (for the duration of an alert). May be an option to consider?

**Reply by quicksi1vr** [reply]
Thank you @hufman and @beer-matt 
I imagined it would be difficult, especially what you mentioned - detecting the alert, and being able to switch audio instantly.
I'll spend some time on it this winter. If I get anywhere, I'll post an update here.
